static/styles/image_upload_widget.css

```diff
@@ -120,6 +133,15 @@
         z-index: 99;
     }
 
+    .image_file_input_error {
```

Comment:
it would be nice if you make `image_file_input_error`  CSS common for user avatar, realm icon

Reply:
Sure ! it would look cleaner.